libs/fluid-elements/tab-group/src/lib/tab-group/tab-group.spec.ts

```diff
@@ -26,9 +26,21 @@ function tick(): Promise<void> {
 describe('Fluid tab group', () => {
```

Comment:
you could add some tests for:
- if the initialization is done correctly, like we have tried in the fluid-dev app.
- if it handles everything correctly if you set `active` on a tab, after it has been initialized
- if it handles everything correctly when tabs are removed / added (slotchange)
- if it deals with it correctly, if there are no longer any tabs in there?
  + Even if the component was interacted with before.